tests/regression-sprint-1.coffee

```diff
@@ -246,7 +246,7 @@ describe '| regression-sprint-1 |', ->
           page.chrome.url (url)->
             url.assert_Contains '/guest/default.html'
             page.open '/user/main.html',(html,$)->
-              $('#loginwall .alert #message').html().assert_Is 'You need to login to see that page.'
+              $('#message').html().assert_Is '<span>You need to login to see that page.</span>'
```

Comment:
Hey @michaelhidalgo can you use **.text()** instead of **.html** for these tests?

That way we avoid having to put html tags inside the assert text